source-academy/cadet-frontend
# Parse program for every breakpoint

### Description

Fixes #1126 .  A little hint is now printed for each breakpoint if it is set at an invalid location. This helps you diagnose (or ignore) the sometimes cryptic messages that are created.

### Type of change


### How to Test

See images in #1126 . Set breakpoints in the middle of statements. Background: breakpoints are implemented by simply adding "debugger;" statements into the program.

### Checklist

Please delete options that are not relevant.
